tools/python/boutiques/descriptor2func.py

```diff
@@ -0,0 +1,27 @@
+from boutiques.util.utils import loadJson
+from boutiques import validate, execute, prettyprint
+import json
+
+
+def function(descriptor, name=None, mode='launch'):
+
+    validate(descriptor)
+    descriptor_json = loadJson(descriptor)
+
+    def f(*args, **kwargs):
+        if mode == 'launch':
```

Comment:
I'd like for the function to be generic enough for us to use either mode without redefining it... any ideas how we can pass this option on the fly?

Reply:
done